ccd-definition/CaseField.json

```diff
@@ -36,7 +36,15 @@
     "CaseTypeID": "PERSONAL_INJURY",
     "ID": "claimant",
     "Label": "Claimant details",
-    "FieldType": "Applicant",
+    "FieldType": "Party",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "defendant",
```

Comment:
ID should be respondent instead of defendant as this is a technical common attribute